### DISCUSSION TITLE
How can we implement the xTTSv2 model on the OpenWebUI interface?

### DISCUSSION THREAD

**Discussion by xyz-rainbow** [open-webui/open-webui#3308] [Q&A]
![image](https://github.com/open-webui/open-webui/assets/164746451/ce77b726-0091-4efc-920c-e2eebc6bb93f)

the TTS (xTTSv2) proyect = https://github.com/coqui-ai/TTS

The Openwebui works on a URL and API key
the tts should work on "openedai-speech-integration" https://docs.openwebui.com/tutorial/openedai-speech-integration/
+LOCAL OPENWEBUI URL: http://localhost:3000/
+LOCAL OPENEDAAITTS: http://localhost:8000/

**Comment by windkwbs** [comment]
me to

**Comment by justinh-rahb** [comment]
If you're doing it within a Docker environment, the base URL would be either:
```
http://<service-name>:<port>/v1
- OR -
http://host.docker.internal:<published_port>/v1
```

And you need _anything_ in the key field, no matter what it is... just as long as it's not an ***empty*** string.
